src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java

```diff
@@ -150,4 +149,4 @@ public void getVideoId_shouldFindAllIds() {
     assertThat(servlet.getVideoId(video7).get()).isEqualTo(id);
     assertThat(servlet.getVideoId(video8).get()).isEqualTo(id);
   }
-}
+}
```

Comment:
there might be a setting in vscode thats automatically deleting these, you should change it automatically insert a newline.

Reply:
I'll look into that, thanks!